collection/Kobold Press; Deep Magic Backgrounds, Feats, & Subclasses.json

```diff
@@ -0,0 +1,11555 @@
+							"colLabels": [
+								"{@dice d6}",
+								"Flaw"
+							],
+							"colStyles": [
+								"col-1 text-center",
+								"col-11"
+							],
+							"rows": [
+								[
+									"1",
+									"I cannot lie outright, only by omission, because of my years with the fey."
+								],
+								[
+									"2",
+									"I will never willingly make another promise to anyone or anything."
+								],
+								[
+									"3",
+									"The fey creature that abducted me haunts me in my dreams."
+								],
+								[
+									"4",
+									"I will never accept free food, drink, or gifts from strangers."
+								],
+								[
+									"5",
+									"I speak in sing-song voice and in rhymes. I never answer questions directly."
+								],
+								[
+									"6",
+									"I am certain that my freedom from the fey is only temporary and that any fey I meet want to recapture me."
+								]
+							]
+						},
+						{
+							"type": "table",
+							"colLabels": [
+								"{@dice d6}",
+								"Ideal"
+							],
+							"colStyles": [
+								"col-1 text-center",
+								"col-11"
+							],
+							"rows": [
+								[
+									"1",
+									"Art. Works of art are meant to question, Challenge, and disturb. (Any)"
+								],
+								[
+									"2",
+									"Pleasure. There are no long-term Rewards, only the enjoyments of the moment. (Chaotic)"
+								],
+								[
+									"3",
+									"Independence. Independence is the privilege of the few who are strong to assert it. (Any)"
+								],
+								[
+									"4",
+									"Fellowship. Lasting fellowship is born in shared suffering and Teamwork when overcoming adversity. (Any)"
+								],
+								[
+									"5",
+									"Self-confidence. There are no views, perspectives, or standards of any worth aside from my own. (Evil)"
+								],
+								[
+									"6",
+									"Malice. I feel nothing but spite, rancor, and envy for those that took me, and for those that let me be taken. (Evil)"
+								]
+							]
+						}
+					]
+				}
+			],
+			"fluff": {
+				"entries": [
+					"Years, possibly centuries, ago you were captured by the fey. Each host, as the fey called themselves, treated their “guest” differently. The Relationship you forged with your captors defines you just as strongly as the circumstances of your abduction and of your Escape or release. You have been indelibly marked—emotionally, psychically, and physically—by the fey and their Special brand of hospitality."
+				]
+			}
+		},
+		{
+			"name": "Soul Channeler",
+			"source": "DM:BFS",
+			"skillProficiencies": [
+				{
+					"arcana": true,
+					"choose": {
+						"from": [
+							"persuasion",
+							"intimidation"
+						],
+						"count": 1
+					}
+				}
+			],
+			"toolProficiencies": [
+				{
+					"alchemist’s supplies": true,
+					"poisoner’s kit": true
+				}
+			],
+			"languageProficiencies": [
+				{
+					"choose": {
+						"from": [
+							"Infernal",
+							"Abyssal"

```

Comment:
My mistake. These (and all languages in "languageProficiencies" objects, for future reference) should also be lowercase.

Reply:
Thank you! 
Its the first time ive made a compendium let alone a JSON file (or even touched coding) so this is all quite new to me.
I'll make that change and apply it, if you need me to change more to make it work please let me know!